functests/09-test-base-image-namespace_rbac.sh

```diff
@@ -0,0 +1,191 @@
+	local status=1
+        for idx in $( seq 1 40); do
+	        (( ${V} >= 1 )) && echo "Attempt ${idx} to find namespace ${TEST_NS}"
+                oc get ns | grep  ${TEST_NS} > /dev/null
+                if [ $? == 0 ]; then
+                        status=0
+	                (( ${V} >= 1 )) && echo "Found Namespace in Attempt ${idx}"
+                        break
+                fi
+	        sleep 3s
+        done
+        echo $status
+}
+
+function find_editclusterrole {
+        # Find if the Edit Clusterrole is created
+        
+	local status=1
+	for idx in $( seq 1 40); do
+	        (( ${V} >= 1 )) && echo "Attempt ${idx} to find clusterrole ${EDIT_ROLE}"
+                oc get clusterrole | grep ${EDIT_ROLE} >  /dev/null
+                if [ $? == 0 ]; then
+                        status=0
+	                (( ${V} >= 1 )) && echo "Found clusterrole in Attempt ${idx}"
+                        break
+                fi
+	        sleep 3s
+        done
+	echo $status
+}
+
+function find_viewrole {
+        # Find if the View Role is created
+
+	local status=1
+	for idx in $( seq 1 40); do
+	        (( ${V} >= 1 )) && echo "Attempt ${idx} to find role ${VIEW_ROLE}"
+                oc get role -n ${TEST_NS} | grep ${VIEW_ROLE} >  /dev/null
+                if [ $? == 0 ]; then
+                        status=0
+	                (( ${V} >= 1 )) && echo "Found role in Attempt ${idx}"
+                        break
+                fi
+	        sleep 3s
+        done
+	echo $status
+}
+
+function find_viewrolebinding {
+        # Find if the View Rolebinding is created
+
+	local status=1
+	for idx in $( seq 1 40); do
+	        (( ${V} >= 1 )) && echo "Attempt ${idx} to find rolebinding ${VIEW_ROLE}"
+                oc get rolebinding -n ${TEST_NS} | grep ${VIEW_ROLE} >  /dev/null
+                if [ $? == 0 ]; then
+                        status=0
+	                (( ${V} >= 1 )) && echo "Found rolebinding in Attempt ${idx}"
+                        break
+                fi
+	        sleep 3s
+        done
+	echo $status
+}
+
+oc create -n ${SSP_OP_POD_NAMESPACE} -f "${SCRIPTPATH}/common-templates-unversioned-cr.yaml" || exit 2
+# TODO: SSP-operator needs to improve its feedback mechanism
+sleep 21s
+
+# Check if the namespace is created
+echo "CNV-4494: SSP Base Image OS namespace is created"
+RET=$(find_namespace)
+
+# Check if the edit clusterrole is created
+echo "CNV-4584: Edit Clusterrole is created"
+if [ $RET == 0 ]; then
+	RET=$(find_editclusterrole)
+fi
+
+# Check if view role is created
+echo "CNV-4772: View Role is created"
+if [ $RET == 0 ]; then
+	RET=$(find_viewrole)
+fi
+
+# Check if view rolebinding is created
+echo "CNV-4772: View Rolebinding is created"
+if [ $RET == 0 ]; then
+	RET=$(find_viewrolebinding)
+fi
+
+# Check if the role, rolebinding, clusterrole and namespace are recreated upon deletion
+oc delete clusterrole ${EDIT_ROLE} || exit 2
+oc delete ns ${TEST_NS} || exit 2
+
+sleep 21s
```

Comment:
I think it would be a better idea to use a loop with a retry count, because the operator's response time can vary